### PR TITLE
Updating bpm suite product overview page.

### PR DESCRIPTION
**Pull request by eschabell**
Same updates pushed to jboss.org project page for bpm suite, see https://github.com/jboss-developer/www.jboss.org/pull/975

### CODE REVIEW

**Comment by eschabell** [comment]
This looks great, please push on through at your convenience.

**Comment by eschabell** [comment]
That would be great.... I can also open up that vimeo album...

File: products/bpmsuite/_common/product.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ has_installer: true
 current_version: 6.1.0.GA
 documentation_minor_version: 6.1
 xpaas_url: https://www.openshift.com/developers/jboss-bpms
-intro_video: https://vimeo.com/95505857
-vimeo_album: http://vimeo.com/album/2982446
-youtube_album: https://www.youtube.com/playlist?list=PLf3vm0UK6HKphPQWLX8SPJlPyqDhpZS4U
+intro_video: https://vimeo.com/142667410
```
Comment (eschabell) [inline products/bpmsuite/_common/product.yml]: This is hosted on redhat's youtube, do you support that? When I tried it did not work so quickest fix is to push the original content to vimeo on my account.

BU product owners (PM, PMM and myself) want very much to update the content here that is very, very outdated.

Can we push this for now and I will push changes once hosted wherever you want it?

Comment (paulrobinson) [inline products/bpmsuite/_common/product.yml]: So did you pull the video (somehow) from Red Hat's Vimeo account and put it on your own? In that case, how about you do the same, but put it onto JBoss Developer's Vimeo account?

Also, if you pass me a link to the original video on Red Hat's Vimeo account I can try and see why the site wouldn't accept it.

**Comment by eschabell** [comment]
Preview link above not working?

**Comment by eschabell** [comment]
Fixed and pushed on https://github.com/redhat-developer/developers.redhat.com/pull/485

**Pull request closed without merging**